src/undo/MergeLayerDownUndoAction.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Xournal++
+ *
+ * Undo action for merging layer down
+ *
+ * @author Xournal++ Team
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "UndoAction.h"
+
+class Layer;
+class LayerController;
+
+class MergeLayerDownUndoAction: public UndoAction {
+public:
+    MergeLayerDownUndoAction(LayerController* layerController, const PageRef& page, Layer* upperLayer,
+                             Layer* lowerLayer, int upperLayerPos);
+
+    // TODO: Did not copy non-default destructor definition:
+    //       There is a non-default destructor defined in
+    //       'src/undo/MoveLayerUndoAction.{h,cpp}' which simply sets all
+    //       pointer members to `nullptr`. Is there any reason to do this?
+    //       All I can think of is assuming that some other part of our program
+    //       still holds a pointer to this destroyed undo/redo action and if its
+    //       access doesn't segfault it might try to access these old non-null
+    //       pointers which could then cause a segfault one level deeper.
+    //       Seems convoluted to me, but maybe I'm missing something?
```

Comment:
I don't think we need a `virtual` destructor here (even though most other undo actions do have a virtual destrouctor).

If the reason for including a virtual destructor was to ensure that `virtual UndoAction::~UndoAction` was called, [it should be fine to omit it](https://stackoverflow.com/a/2198400/17055750).